### DISCUSSION TITLE
really safe?

### DISCUSSION THREAD

**Discussion by rs-neilchuang** [Ylianst/MeshCentral#6313] [Issues]
As the title suggests, just want to make sure it's safe to use , thanks guys!
The link below shows the results of the virustotal scan.

https://www.virustotal.com/gui/file/1033b88d155a6fa413d8fc1156cc873c393c591857c98ea0e4b7cf520c2b6528?nocache=1

**Comment by si458** [comment]
Meshcentral is perfectly safe!
Unfortunately, due to bad actors abusing meshcentral for malicious reasons, most antivirus software have marked it now as a virus/hacktool :( 
The only thing u can do is if it's for corporate use or ur own personal use, is to allow the software in your antivirus program
OR
get a code-signing certificate then customise the exe and HOPE antiviruses are happy with it!

**Reply by JSuenram** [reply]
Codesigning does help a lot here... we signed our Agents 2 years ago and since then never had an issue again.
However we used a more expensive cert which says it does pass Windows Smartscreen instant, and it works.....

**Reply by inzi** [reply]
Same here, code signed agent only nags when setting up, but otherwise works without issue that I've seen.
